.github/workflows/conan.yml

```diff
@@ -40,6 +40,63 @@ jobs:
           github.event_name == 'schedule' &&
           steps.git_info.outputs.current_commit == steps.last_successful_commit.outputs.commit-hash
 
+  build_windows:
+    if: github.repository_owner == 'viamrobotics'
+    needs: [prepare]
+    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - target: x86_64-windows
+            platform: windows_x86_64
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v4
+
+      - name: Install cmake
+        uses: ssrobins/install-cmake@v1
+        with:
+          version: 3.27.2
+
+      - name: Install python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.12'
+
+      - name: Install Scoop
+        uses: MinoruSekine/setup-scoop@v4.0.1
+
+      - name: install buf 'n stuff
+        run: scoop install buf ninja
+
+      - name: Install dependencies
+        uses: crazy-max/ghaction-chocolatey@v3
+        with:
+          args: install -y conan git 7zip jq
```

Comment:
Do you actually need 7zip or jq for this?

Reply:
probably not! I'll get rid of it.